### PR TITLE
add parameters for sheffield hawk

### PR DESCRIPTION
**Pull request by bobturneruk**
Please can you merge my species?

### CODE REVIEW

File: params/sheffield_hawk.R [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 4.9
+    
+# e.g. "anas_krystallinus"    
+species.name <- "sheffield_hawk"
```
Comment (EdwinB12) [inline params/sheffield_hawk.R]: Could you please change the name of your species?

**Review by EdwinB12 — CHANGES_REQUESTED** [review]
Could you please change the name of your species.

File: params/sheffield_hawk.R (hunk 1 above)
Comment (bobturneruk) [reply params/sheffield_hawk.R]: Good spot - I've changed the name.

Comment (EdwinB12) [reply params/sheffield_hawk.R]: Thanks Bob, that's a great name!

**Review by EdwinB12 — APPROVED** [review]

**Pull request merged**